### PR TITLE
Feature: Stats card: Show merged PRs count and percentage

### PR DESCRIPTION
**Pull request by qwerty541**
This pull request adds ability to show count and percentage of merged PRs on stats card using `&show=` query option.

image

I'm not sure about merged PRs percentage icon, what do you think @rickstaa?

### CODE REVIEW

**Review by qwerty541 — COMMENTED** [review]
@rickstaa I committed an alternate icon for "Merged PRs Percentage" stat. Let me know which of them better on your opinion.

image

File: readme.md [hunk 1]
```diff
@@ -156,10 +156,10 @@ You can pass a query parameter `&hide=` to hide any specific stats with comma-se
 
 You can pass a query parameter `&show=` to show any specific additional stats with comma-separated values.
 
-> Options: `&show=reviews,discussions_started,discussions_answered`
+> Options: `&show=reviews,discussions_started,discussions_answered,prs_merged,merged_prs_percentage`
```
Comment (rickstaa) [inline readme.md:159]: Maybe for consistency, we can call it `prs_merged, prs_merged_percentage`. I am, however, also good with your current names if you think they are more apparent.

**Review by rickstaa — APPROVED** [review]
Apart from the naming, I think this pull request is clean 🚀! Feel free to merge it if you want.

File: readme.md (hunk 1 above)
Comment (qwerty541) [reply readme.md]: I have update the code following your suggestion. Maybe it's better, but it hard for me to judge since i'm not english native speaker.

**Pull request merged**